core/io/logger.h

```diff
@@ -31,10 +31,16 @@
 #ifndef LOGGER_H
 #define LOGGER_H
 
+#include "modules/modules_enabled.gen.h" // For regex.
+
 #include "core/io/file_access.h"
 #include "core/string/ustring.h"
 #include "core/templates/vector.h"
 
+#ifdef MODULE_REGEX_ENABLED
+#include "modules/regex/regex.h"
+#endif // MODULE_REGEX_ENABLED
+
```

Comment:
This is the canonical style for this combo:
```suggestion
#include "core/io/file_access.h"
#include "core/string/ustring.h"
#include "core/templates/vector.h"

#include "modules/modules_enabled.gen.h" // For regex.
#ifdef MODULE_REGEX_ENABLED
#include "modules/regex/regex.h"
#endif // MODULE_REGEX_ENABLED

```